health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -98,4 +101,133 @@ export const UICustomizations = {
     },
   },
 
+  UserManagementConfig: {
+
+    customValidationCheck: (data) => {
+      const { phone } = data;
+      const mobileRegex = /^[0-9]{10}$/;
+      debugger
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)